### DISCUSSION TITLE
why is the dist folder commited to the repo??

### DISCUSSION THREAD

**Discussion by grinish21** [golangci/golangci-lint-action#1080] [Q&A] [answered]
Just out of curiousity...
Is there a specific reason why the dist folder is committed into the repo?? isn't `dist` a build artifact does it need to be in the repo?? 

I found a PR where it was actually updated via commit -> https://github.com/golangci/golangci-lint-action/pull/160

**Comment by ldez** [comment, marked as answer]
Because it's a GitHub Action written in Typescript, to work, the executable code must be in JavaScript.

Note: all the PRs that change the code (and some dependencies) always contain an update of the `dist` folder.

**Reply by grinish21** [reply]
Thank you for the time and apologies for such a stupid question. I should have realized it.
